modules/user_manual/pages/files/access_webdav.adoc

```diff
@@ -463,17 +453,15 @@ certificate.
 [[solution-5]]
 === Solution
 
-____________________________________________________________________________________________________________________________________________________________________
+""
```

Comment:
The list doesn't need surrounding air quotes. I believe it read better without them.